src/components/TodoFilterLink/TodoFilterLink.tsx

```diff
@@ -0,0 +1,30 @@
+import { Status } from '../../types/enums';
+import cn from 'classnames';
+import { useTodos } from '../context/TodosContext';
+import { useCallback } from 'react';
+
+type Props = {
+  statusValue: Status;
+  children: React.ReactNode;
+};
+export const TodoFilterLink: React.FC<Props> = ({ statusValue, children }) => {
+  const { statusTodo, setStatusTodo } = useTodos();
+
+  const handleClickFilter = useCallback(() => {
```

Comment:
You don't need to wrap with `useCallback` here as this function is used only in this component